src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -14040,6 +14016,89 @@ static CustomHandler F3()
                 Diagnostic(ErrorCode.ERR_EscapeVariable, "h3").WithArguments("h3").WithLocation(29, 16));
         }
 
+        [WorkItem(63306, "https://github.com/dotnet/roslyn/issues/63306")]
+        [Theory]
+        [InlineData(LanguageVersion.CSharp10)]
+        [InlineData(LanguageVersion.CSharp11)]
+        public void RefEscape_13A(LanguageVersion languageVersion)
+        {
+            var code =
+@"using System.Runtime.CompilerServices;
+[InterpolatedStringHandler]
+ref struct CustomHandler
+{
+    public CustomHandler(int literalLength, int formattedCount) { }
+    public void AppendFormatted(in int i) { }
+}
+class Program
+{
+    static CustomHandler F1()
+    {
+        int i = 1;
+        return $""{i}""; // 1
```

Comment:
Does this mean that guidance that we just gave the BCL (that only ref struct parameters need to be marked `scoped`) isn't correct?

Reply:
The general guidance is any `ref` parameter or `ref struct` parameter should be `scoped` if it's not returned directly or indirectly from the method.